index.js

```diff
@@ -1,9 +1,17 @@
 'use strict';
 var got = require('got');
-var registryUrl = require('registry-url');
+var npmKeywordUrl = require('npm-keyword-url');
 
-module.exports = function (keyword, cb) {
-	got(module.exports.url(keyword), function (err, data) {
+module.exports = function (opts, cb) {
+	if (typeof opts === 'string') {
+		opts = {keyword: opts};
```

Comment:
if you are using opts then you are already editing your source code and use the updated syntax

```
{
  keyword: 'xyz',
  description: false
}
```

updated anyway